server/handlers/walletHandler/index.js

```diff
@@ -161,6 +162,8 @@ const walletPost = async (req, res) => {
     );
   }
 
+  await queueService.sendWalletCreationNotification(returnedWallet);
```

Comment:
QueueService is a class, it should be initialised first.

const queueService = new QueueService()
await queueService.sendWalletCreationNotification(returnedWallet)